core/arch/arm/plat-imx/drivers/imx_caam.c

```diff
@@ -109,10 +125,10 @@ static TEE_Result caam_get_mkvb(uint8_t *dest)
 	cache_operation(TEE_CACHEFLUSH, &mkvb.jr,
 			sizeof(mkvb.jr.inring[0]));
 
-	/*  Tell CAAM that one job is available */
+	/* Tell CAAM that one job is available */
 	io_write32((vaddr_t)&mkvb.ctrl->jrcfg[MKVB_JR].irjar, 1);
 
-	/*  Busy loop until job is completed */
+	/* Busy loop until job is completed */
```

Comment:
must reset `ret` to an error code (`ret = TEE_ERROR_SECURITY`) since storage of `mkvb_init_jr()` return code.